templates/views/team.html

```diff
@@ -0,0 +1,162 @@
+                    <div class="__shecodeteammember_img">
+                        <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-4 col-sm-6 col-xs-12">
+                <div class="__shecodeteammember_box" data-toggle="modal" data-target="#team_modal">
+                    <div class="__shecodeteammember_img">
+                        <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-4 col-sm-6 col-xs-12">
+                <div class="__shecodeteammember_box" data-toggle="modal" data-target="#team_modal">
+                    <div class="__shecodeteammember_img">
+                        <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-4 col-sm-6 col-xs-12">
+                <div class="__shecodeteammember_box" data-toggle="modal" data-target="#team_modal">
+                    <div class="__shecodeteammember_img">
+                        <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-4 col-sm-6 col-xs-12">
+                <div class="__shecodeteammember_box" data-toggle="modal" data-target="#team_modal">
+                    <div class="__shecodeteammember_img">
+                        <img src="/images/hair-2537564_1920.png" alt="" class="img-responsive">
+                    </div>
+                    <div class="__shecodeteammember_text">
+                        <h5 class="name">Ada Nduka, OYOM</h5>
+                        <p class="role">Founder</p>
+                    </div>
+                </div>
+            </div>
+
+
+        </div>
+    </div>
+
+    <!-- Modal -->
+    <div id="team_modal" class="modal fade team_modal" tabindex="-1" role="dialog" aria-labelledby="contactModalLabel" aria-hidden="true">
```

Comment:
Can this team modal be vertically centered on the page as well? I think it will look better than having it just right at the top

Reply:
ok